### PR TITLE
Validate and process email addresses for user registration

### PR DESCRIPTION
**Pull request by jgabitto**
- Closes https://github.com/cityofaustin/atd-data-tech/issues/6724
- Added email and tolowercase to email key of yup object shape

**How to Test**
- Visit moped site from netlify preview
- Make sure you add/remove user privileges
- Go to staff page and click on the add staff button
- Fill out form and save

### CODE REVIEW

**Review by mateoclarke — APPROVED** [review]
Looks great!! And no regex required  🙌 🚢 

Side note: Can you update the title of this PR to a little more Release Notes user friendly?

**Review by amenity — APPROVED** [review]
Works great! Thanks.

File: moped-editor/src/views/staff/StaffForm.js [hunk 1]
```diff
@@ -77,7 +77,11 @@ const staffValidationSchema = isNewUser =>
     title: yup.string().required(),
     workgroup: yup.string().required(),
     workgroup_id: yup.string().required(),
-    email: yup.string().required(),
+    email: yup
+      .string()
+      .required()
+      .email()
+      .lowercase(),
```
Comment (sergiogcx) [inline moped-editor/src/views/staff/StaffForm.js:84]: Smooth! I am liking this yup library

**Review by sergiogcx — APPROVED** [review]
Cool stuff!

**Pull request merged**